src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -463,9 +464,11 @@ func (cfg Config) cloudPollSleepTime() time.Duration {
 }
 
 func fetchCloudConfig(url string) ([]byte, error) {
-	req, err := http.NewRequest("GET", url, nil)
+	cb := "?" + string(random.Uint32())
```

Comment:
I would use a UUID here to prevent collisions.  Due to the birthday paradox, a 32-bit value has a surprisingly high [chance of collision](http://preshing.com/20110504/hash-collision-probabilities/).